### PR TITLE
Lydia/rs 52 custom simple 404 500 page

### PR DESCRIPTION
**Pull request by LydiaKoil**
image

### CODE REVIEW

File: app/views/errors/internal_server.html.haml [hunk 1]
```diff
@@ -0,0 +1 @@
+= render 'error_message'
```
Comment (danielpaul) [inline app/views/errors/internal_server.html.haml:1]: the wording for this page would be different? error code would be 500, the title would be "Oops! Something went wrong...", description: "Sorry, it's not you. It's us. Our devs have been notified of this issue and are working to fix this issue. Please try again later."

File: not_found.html.erb
Comment (danielpaul) [inline not_found.html.erb:1]: why is there this empty file?

File: spec/requests/errors_spec.rb
Comment (danielpaul) [inline spec/requests/errors_spec.rb:1]: don't need this file if we don't have any valid tests

**Review by danielpaul — CHANGES_REQUESTED** [review]

File: app/views/errors/internal_server.html.haml (hunk 1 above)
Comment (danielpaul) [reply app/views/errors/internal_server.html.haml:1]: @LydiaKoil I don't see this being resolved in the file changes? But I was able to have a look int the commit and that's not what we are looking for. 

* 404 error -> the copy that we had before - this is food
* 500 error -> the new copy I gave

Both the error codes have different copy. They won't both render the same error.

**Review by danielpaul — CHANGES_REQUESTED** [review]

File: app/views/errors/internal_server.html.haml (hunk 1 above)
Comment (LydiaKoil) [reply app/views/errors/internal_server.html.haml:1]: ah ok I get it now

**Review by danielpaul — APPROVED** [review]

**Pull request merged**